src/objects/logo-group/logo-group.stories.mdx

```diff
@@ -1,29 +1,63 @@
 import { Canvas, Meta, Story } from '@storybook/addon-docs';
-// The '!!raw-loader!' syntax is a non-standard, Webpack-specific, syntax.
-// See: https://github.com/webpack-contrib/raw-loader#examples
-// For now, it seems likely Storybook is pretty tied to Webpack, therefore, we are
-// okay with the following Webpack-specific raw loader syntax. It's better to leave
-// the ESLint rule enabled globally, and only thoughtfully disable as needed (e.g.
-// within a Storybook docs page and not within an actual component).
-// This can be revisited in the future if Storybook no longer relies on Webpack.
-// eslint-disable-next-line @cloudfour/import/no-webpack-loader-syntax
-import defaultDemoSource from '!!raw-loader!./demo/demo.twig';
-import defaultDemo from './demo/demo.twig';
+import demo from './demo/demo.twig';
 import logos from './demo/logos.json';
+const justifyOptions = ['start', 'center', 'end'];
+const demoTemplateSource = (_src, storyContext) => {
+  const args = storyContext.args || storyContext.initialArgs;
+  const twigArgs =
+    Object.keys(args).length > 0
+      ? ` with ${JSON.stringify(args, null, 2)}`
+      : '';
+  return `{% embed '@cloudfour/objects/logo-group/logo-group.twig'${twigArgs} %}
+  {% block content %}
+    {# logos #}
+  {% endblock %}
+{% endembed %}`;
+};
```

Comment:
I found the raw demo source was _mostly_ noise, so I made a custom template source function to be more helpful.

When we eventually update the project to the latest Storybook and Vite, I honestly think we should consider starting over when it comes to the source previews since this sort of thing is so widespread. But for now, it's nice to have a useful code sample.

Reply:
I feel like this always ends up being a hard challenge when writing docs. Hopefully we can find a better pattern after the upgrade :+1: